NEWS.md

```diff
@@ -1,3 +1,11 @@
+CohortDiagnostics 2.2.2
```

Comment:
Is this intended to be a hotfix for the main branch? Develop includes code for version 3.0.0.

Considering that this is the removal of functionality (though most users probably aren't using it), I suggest it goes in to version 3.